pkgs/development/tools/azure-functions-core-tools/default.nix

```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp -prd *.dll *.so gozip func $out/bin
+    cp -prd * gozip func $out/bin
```

Comment:
Was this changed to be less granular for a reason? Furthermore, couldn't the explicit `gozip` and `func` arguments be removed since the wildcard matches everything?

Reply:
@winterqt yes the change to less granular was because when I was using command `func` it failed because it needed some of the files that was excluded. Good point that `gozip` and `func` can be removed, since wildcard. I will update the code.

Reply:
Just out of curiosity what was failing? Unfortunately I don't work with this tool much anymore so I haven't had much time to troubleshoot/validate updates to this package. Last I had worked on it there was an issue where it was referencing a hard coded path to `bash` that was causing some issues with I think push functions to azure but its been a while since I've used..

Reply:
Sorry @jshcmpbll, at the moment I do not remember exactly what was falling, made the PR in june.  Should have written exactly what was failing, before I forgot. But I remember that copying over missing file solved the problem.